### PR TITLE
Adding 'Getting Started FAQ' to README.md

### PR DESCRIPTION
**Pull request by kbtbc**
It's not a Wiki yet, but it's what I've got. 

Will certainly save on some support redundancy.

Feedback and modification is welcome.

Cheers!
-BigFun

### CODE REVIEW

**Comment by Noctem** [comment]
Could you add this to the [wiki](https://github.com/Noctem/pokeminer/wiki) instead and just put a link to it in the readme?

**Comment by kbtbc** [comment]
I tried to do a pr to the Wiki initially, but github didn't like it.  I think I need permission.

File: README.md [hunk 1]
```diff
@@ -90,6 +90,10 @@ The gyms statistics server is in a separate file, because it's intended to be sh
 
 [![gyms](https://i.imgur.com/MWpHAEWm.jpg)](pokeminer/static/demo/gyms.png)
 
+## Getting Started Tips & FAQs
+
+Visit our [Wiki](https://github.com/Noctem/pokeminer/wiki) for more info.
```
Comment (Noctem) [inline README.md]: Could you update the URL to <https://github.com/Noctem/Monocle/wiki>?

Comment (kbtbc) [inline README.md]: done

**Pull request merged**